tdh/resource_cluster.go

```diff
@@ -544,13 +531,11 @@ func (r *clusterResource) Update(ctx context.Context, req resource.UpdateRequest
 			Id:            state.ID.ValueString(),
 			TargetVersion: plan.Upgrade.TargetVersion.ValueString(),
 			RequestType:   "SERVICE",
-			Metadata:      upgrade_service.UpdateClusterVersionRequestMetadata{OmitBackup: omitBackup},
+			Metadata:      upgrade_service.UpdateClusterVersionRequestMetadata{OmitBackup: strconv.FormatBool(omitBackup)},
```

Comment:
Can't we change omitBackup type in the schema and model?

Reply:
Addressed